mobilecoind/src/service.rs

```diff
@@ -4848,6 +5130,87 @@ mod test {
         }
     }
 
+    #[test_with_logger]
+    /// Should return a correct proof-of-membership for each requested TxOut.
+    fn test_get_tx_out_results_by_pub_key(logger: Logger) {
+        let mut rng: StdRng = SeedableRng::from_seed([23u8; 32]);
+
+        let sender = AccountKey::random(&mut rng);
+        let data = MonitorData::new(
+            sender.clone(),
+            0,  // first_subaddress
+            20, // num_subaddresses
+            0,  // first_block
+            "", // name
+        )
+        .unwrap();
```

Comment:
:upside_down_face: A bit of a nit but I would avoid these kind of inline argument comments, if the arguments get renamed these comments are likely to be missed.  It's unfortunate that MonitorData::new() has 3 u64 arguments right next to each other and maybe that's why these comments are here to account for that
```suggestion
        let data = MonitorData::new(sender.clone(), 0, 20, 0, "").unwrap();
```